apps/aecore/test/aecore_sync_SUITE.erl

```diff
@@ -511,12 +511,11 @@ new_tx(#{node1 := N1, node2 := N2, amount := Am, fee := Fee} = M) ->
                     [ [<<"http">>, <<"internal">>, <<"port">>],
                       aehttp, [internal, port], 8143], 5000),
     Params = #{sender_pubkey => PK1,
```

Comment:
Since this is the internal API and sender is not required - why do we send it at all? I see we are sending the raw binary...

Reply:
I've just used the new HTTP client here. I keep all the params as they were before.